<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowStatisticsInsDiscountRegistFee</class>
 <widget class="QMainWindow" name="WindowStatisticsInsDiscountRegistFee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>掛號費優待統計</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableWidget_medical_record">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_week">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>300</height>
               </size>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <column>
               <property name="text">
                <string>項目</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>一</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>二</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>三</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>四</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>五</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>星</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>期</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>六</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>合計</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_chart"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <action name="action_export_excel">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-export.svg</normaloff>../icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>匯出Excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
